src/image-marker.js

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'clappr'
+import StandardMarker from './standard-marker'
+
+/*
+ * An implementation of an image Marker, which can show an image.
+ */
+export default class ImageMarker extends StandardMarker {
+
+  /*
+   * time: the time in seconds that this marker represents
+   * tooltipImage: the image to be shown (optional)
+   * width: width for the image (Default: 200px)
+   * height: height for the image (Default: auto)
+   */
+  constructor (time, tooltipImage, width, height) {
+    super(time, tooltipImage)
+    this._tooltipText = tooltipImage || null
```

Comment:
yes tried it. didn't like copy paste approach. perhaps we can rename the variable to a generic one like tooltipContent and call it a day?